source/vibe/web/common.d

```diff
@@ -305,8 +321,10 @@ MethodAttribute method(HTTPMethod data)
 	
 	shared static this()
 	{
+		// Tie IAPI.getInfo to "GET /root/foo/info2"
 		registerRestInterface!IAPI(new URLRouter(), new API(), "/root/");
-		// now IAPI.getInfo is tied to "GET /root/info2"
+		// Or just to "GET /foo/info2"
```

Comment:
Is the "/root" above ignored when `@path` is given? I think the URL should be `/root/foo/info2` when both are given like here.